python/vyos/frr.py

```diff
@@ -210,9 +247,15 @@ def reload_configuration(config, daemon=None):
     LOG.debug(f'reload_configuration: Executing command against frr-reload: "{cmd}"')
     output, code = popen(cmd, stderr=STDOUT)
     f.close()
+    emsg = ''
     for i, e in enumerate(output.split('\n')):
```

Comment:
I'm not sure if I understand the logic here. As I understand, the idea is to relay the FRR error message to the user.
But why is it limited to a small subset of selected error messages? When they add more messages (and I'm sure they will), that code will not consider them errors and will produce misleading outputs.

From https://github.com/FRRouting/frr/blob/master/bgpd/bgp_vty.c#L978 it looks like FRR does exit with a specific non-zero exit code on those errors — is there a reason why we can't check the exit code and if it's not zero, just log the complete stderr?

Reply:
Done, now we return the whole frr log